docs/tutorials/create_architecture.md

```diff
@@ -1,221 +1,223 @@
-# 評価準備
+# Preparation for measuring latency of callback-chains
 
-CARET は、評価する際にアーキテクチャファイルの作成が必要です。
+CARET is capable of measuring latency of not only callback functions and topic communications, but also node chains.
+Specifying a callback function as target is very simple because name is corresponding to the callback function. Topic communication is specified by a topic name, a source node, and a destination node. However, specifying a node chain is more difficult than them. A node chain is defined as a combination of callback functions and topic communications, and often called as "path". Though node chain is regarded as DAG (Direct Acyclic Graph), a node chain may have loops and forks. Loops and forks make a node chain make complicated. Additionally, a large system tends to have many candidates of node chains because it is constructed of many callback functions which are communicated via many topics.
```

Comment:
Loops and forks make a node chain ~~make~~ complicated.

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/36